LICENSE

```diff
@@ -2,10 +2,27 @@ The SGGW license (SGGW)
 
 Copyright (c) 2023 SGGW
 
-Entire repository is owned by SGGW (Warsaw University of Life Sciences)
+Entire repository is owned by SGGW (Warsaw University of Life Sciences) and is associated with the subject "Software engineering"
```

Comment:
we must point out that it is created as part of the course `name` by students of SGGW 2021/2024 or something like that.